styleguide/derek/pattern-components/headers/header-refresh.ejs

```diff
@@ -6,19 +6,17 @@ if(!desc) { var desc = "The cloud is a mystery wrapped in a riddle. We can unwra
   <div class="refreshBannerCombined-container">
     <div class="refreshBannerCombined-svgContainer">
       <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-        width="1442px" height="600px" viewBox="0 0 1442 600" style="enable-background:new 0 0 1442 777;"
+        width="1442px" height="500px" viewBox="0 0 1442 500" style="enable-background:new 0 0 1442 677;"
         class="refreshBannerCombined-svg" xml:space="preserve">
         <style type="text/css">
-          .leftSide{opacity:0.8;fill:#AB2D34;}
-          .bottomSide{opacity:0.8;fill:#732126;}
-          .textBox{fill:#FFFFFF;}
+          .leftSide{opacity:0;fill:#AB2D34;}
+          .bottomSide{opacity:0;fill:#732126;}
```

Comment:
If we're not displaying `.leftSide` or `.bottomSide` we should just remove the polygon's from the svg altogether.  No sense in having it and hiding it.